panic when moving vendor dir

```
[INFO]	Replacing existing vendor dependencies
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x20 pc=0x490cce]

goroutine 1 [running]:
panic(0x9857c0, 0xc8200120c0)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/Masterminds/glide/vendor/github.com/codegangsta/cli.HandleAction.func1(0xc8204ef3c0)
	/go/src/github.com/Masterminds/glide/vendor/github.com/codegangsta/cli/app.go:472 +0x417
panic(0x9857c0, 0xc8200120c0)
	/usr/local/go/src/runtime/panic.go:443 +0x4e9
github.com/Masterminds/glide/repo.(*Installer).Export(0xc820184ec0, 0xc82022a540, 0x0, 0x0)
	/go/src/github.com/Masterminds/glide/repo/installer.go:364 +0xafe
github.com/Masterminds/glide/action.Install(0xc820184ec0, 0xa40800)
	/go/src/github.com/Masterminds/glide/action/install.go:59 +0x4e5
main.commands.func12(0xc8201943c0, 0x0, 0x0)
	/go/src/github.com/Masterminds/glide/glide.go:544 +0x492
reflect.Value.call(0x8cbe80, 0xb225b0, 0x13, 0xa278d0, 0x4, 0xc8204ef320, 0x1, 0x1, 0x0, 0x0, ...)
	/usr/local/go/src/reflect/value.go:435 +0x120d
reflect.Value.Call(0x8cbe80, 0xb225b0, 0x13, 0xc8204ef320, 0x1, 0x1, 0x0, 0x0, 0x0)
	/usr/local/go/src/reflect/value.go:303 +0xb1
github.com/Masterminds/glide/vendor/github.com/codegangsta/cli.HandleAction(0x8cbe80, 0xb225b0, 0xc8201943c0, 0x0, 0x0)
	/go/src/github.com/Masterminds/glide/vendor/github.com/codegangsta/cli/app.go:481 +0x2ee
github.com/Masterminds/glide/vendor/github.com/codegangsta/cli.Command.Run(0xa2b220, 0x7, 0xa2b0b8, 0x1, 0x0, 0x0, 0x0, 0xaa9460, 0x20, 0x0, ...)
	/go/src/github.com/Masterminds/glide/vendor/github.com/codegangsta/cli/command.go:186 +0x1301
github.com/Masterminds/glide/vendor/github.com/codegangsta/cli.(*App).Run(0xc820001500, 0xc82000a0f0, 0x3, 0x3, 0x0, 0x0)
	/go/src/github.com/Masterminds/glide/vendor/github.com/codegangsta/cli/app.go:235 +0xa9c
main.main()
	/go/src/github.com/Masterminds/glide/glide.go:128 +0x855
```

--- a/repo/installer.go
+++ b/repo/installer.go
@@ -361,7 +361,7 @@ func (i *Installer) Export(conf *cfg.Config) error {
 
 	// When there are different physical devices we cannot rename cross device.
 	// Fall back to manual copy.
-	if strings.Contains(err.Error(), "invalid cross-device link") {
+	if err != nil && strings.Contains(err.Error(), "invalid cross-device link") {
 		msg.Debug("Cross link err, trying manual copy: %s", err)
 
 		err = gpath.CopyDir(vp, i.VendorPath())

